ContentProviderPagingKotlin/app/src/main/kotlin/com.example.android.contentproviderpaging/ImageProvider.kt

```diff
@@ -79,7 +79,7 @@ class ImageProvider : ContentProvider() {
         }
 
         val maxIndex = Math.min(offset + limit, files.size)
-        for (i in offset..maxIndex) {
+        for (i in offset..maxIndex - 1) {
```

Comment:
```suggestion
        for (i in offset until maxIndex) {
```

Reply:
Thank your for your suggestion. Helped me lot about Kotlin syntax. Still learning :)